content/references/data-api.md

```diff
@@ -3088,19 +3055,17 @@ A successful response uses the HTTP code **200 OK** and has a JSON body with the
 
 Daily Validator Report Object fields:
 
-| Field                   | Value                           | Description      |
-|:------------------------|:--------------------------------|:-----------------|
-| `validation_public_key` | String - Base-58 [Public Key][] | This validator's validator public key. |
-| `date`                  | String - [Timestamp][]          | The start time of the date this object describes. |
-| `total_ledgers`         | Integer                         | Number of ledger indexes for which this validator submitted validation votes. If this number is much lower than other validators in the same time period, that could mean the validator had downtime. |
-| `main_net_agreement`    | [String - Number][]             | The fraction of consensus-validated production network ledger versions for which this validator voted in this interval. `"1.0"` indicates 100% agreement. |
-| `main_net_ledgers`      | Integer                         | Number of consensus-validated [production network](parallel-networks.html) ledger versions this validator voted for. |
-| `alt_net_agreement`     | [String - Number][]             | The fraction of the consensus-validated [Test Network](parallel-networks.html) ledger versions this validator voted for. `"1.0"` indicates 100% agreement. |
-| `alt_net_ledgers`       | Integer                         | Number of consensus-validated [Test Network](parallel-networks.html) ledger versions this validator voted for. |
-| `other_ledgers`         | Integer                         | Number of other ledger versions this validator voted for. If this number is high, that could indicate that this validator was running non-standard or out-of-date software. |
-| `last_datetime`         | Integer                         | The last reported validation vote signed by this validator. |
-| `domain`                | String                          | (May be omitted) The DNS domain associated with this validator. |
-| `domain_state`          | String                          | The value `verified` indicates that this validator has a [verified domain](run-rippled-as-a-validator.html#domain-verification) controlled by the same operator as the validator. The value `AccountDomainNotFound` indicates that the "Account Domain" part of Domain Verification is not set up properly. The value `RippleTxtNotFound` indicates that the ripple.txt step of Domain Verification is not set up properly. |
+Each member in the `reports` array is a Single Validator Report Object with data on that validator's performance on that day. Each has the following fields:
```

Comment:
Consider adding a `#### Daily Validator Report Object` heading above this paragraph, just to match style/structure used for other methods?